DECISION-MAKING.md

```diff
@@ -0,0 +1,151 @@
+`Last update: 2018-07`
+
+# Service owners
+
+High level service and product owners for Mozilla IAM.
+
+* Capability owner: @jeffbryner
+  * Product owner: @gdestuynder
+    * Auth0 & Auth0 Lock & Auth0 Dashboard: @jdow @akrug @gdestuynder
+    * Service Integration (CIS): @akrug
+    * LDAP: @jdow
+    * DuoSecurity: @gdestuynder
+    * Mozillians.org: @hmitsch
+    * WorkDay: HRIS
+
+## Architects
+
+* @gdestuynder (IAM, Security)
+* @mvk-mozilla (IT)
+
+## Program Managers
```

Comment:
Maybe worth adding an explicit UX section with assignment to Megan? (rather than UX as a role in each product)

Reply:
GitHub did not detect it, but done as well